assets/css/startertheme.css

```diff
@@ -0,0 +1,123 @@
+.project-grid {
+  margin-left: 100px;
+  margin-right: 100px;
+  margin-top: 450px;
+}
+
+.project-container .project-image {
+  position: relative;
+  margin: auto;
+  width: 100%;
+  min-width: 220px;
+  max-width: 350px;
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-position: center center;
+  background-size: cover;
+  border-radius: 10px;
+  border: 2.5px solid #525252;
+  box-sizing: border-box;
+  overflow: visible;
+}
+
+.project-container .project-image:before {
+  content: "";
+  display: block;
+  padding-top: 100%;
+  position: relative;
+  background: rgba(255, 255, 255, 0.75);
+  opacity: 0;
+  border-radius: 10px;
+}
+
+.project-container .project-image:after {
+  content: "Learn More";
+  font-family: "Quicksand", sans-serif;
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  color: #333;
+  font-size: 2rem;
+  font-weight: 400;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0;
+}
+
+.project-container .project-image:hover:before {
+  opacity: 1;
+  transition: all 200ms ease-in-out;
+}
+
+.project-container .project-image:hover:after {
+  opacity: 1;
+  transition: all 200ms ease-in-out;
+}
+
+.project-container .project-image .project-description {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: #F9F9F9;
+  border-radius: 10px;
+}
+
+.text-black, .text-black-light, .text-lead, .text-black-secondary, .text-light-secondary, .text-description{
+  font-family: 'Quicksand', sans-serif;
+  font-style: normal;
+  color: #525252 !important;
+}
+
+.text-black {
+  font-size: 24px;
+  line-height: 36px;
+}
+
+.text-black-light {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.font-weight-bold {
+  font-weight: bold !important;
+}
+
+.text-black-secondary {
+  font-size: 36px;
+}
+
+.text-light-secondary {
+  font-size: 18px;
+}
+
+.text-description {
+  font-size: 20px;
+}
+
+.text-lead {
+  font-size: 24px;
+  margin-bottom: 1rem;
+}
+
+.card {
+  background: #F9F9F9 !important;
+  border: 2px solid #525252 !important;
+  border-radius: 10px !important;
+}
+
+.closeIcon {
+  font-size: 2.5em;
+  position: absolute;
+  right: 15px;
+  top: 0px;
+  padding: 10px 20px;
+}
+
+.closeIcon:hover {
+  color: #FF6969;
+  cursor: pointer;
+}
```

Comment:
Same as for the rest of the file, for padding, margins, etc. try using `rem` instead of `px`. Look for bootstrap classes that already implement these before writing custom CSS.